agent/grpc-external/services/peerstream/subscription_manager.go

```diff
@@ -239,16 +243,10 @@ func (m *subscriptionManager) handleEvent(ctx context.Context, state *subscripti
 
 		pending := &pendingPayload{}
 
-		// Directly replicate information about our mesh gateways to the consuming side.
-		// TODO(peering): should we scrub anything before replicating this?
-		if err := pending.Add(meshGatewayPayloadID, u.CorrelationID, csn); err != nil {
```

Comment:
We were never actually sent these to the peer, the handler for these is a no-op in `stream_resources.go`. That case statement has been removed now as well.